app/views/layouts/_header.html.slim

```diff
@@ -9,20 +9,20 @@ nav.navbar.navbar-inverse.navbar-fixed-top role="navigation"
       a.navbar-brand href=root_path  SpacialDB
     #navbar.collapse.navbar-collapse
       ul.nav.navbar-nav
-        li.active
+        li class=('active' if controller_name == 'home')
```

Comment:
You can refactor this to use the `current_page?` helper
http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-current_page-3F

Even better would be to create nav_bar helper that take's the link path and name and works out the class internally see for a nice example: http://solvedstack.com/questions/best-way-to-add-current-class-to-nav-in-rails-3

Let me know if you need any help